modules/books.js

```diff
@@ -0,0 +1,7 @@
+export default class Bookdata {
+  constructor(id, title, author) {
+    this.Id = id;
+    this.Title = title;
+    this.Author = author;
```

Comment:
- [ ] Javascript variables are named following **[camelCase](https://www.syncfusion.com/blogs/post/10-javascript-naming-conventions-every-developer-should-know.aspx)** convention. With this convention, `this.Id` would be `this.id`. Please stick to this convention when naming functions & variables. code written following convention is clear & easy to read. 😄